test/e2e_node/cpu_manager_test.go

```diff
@@ -129,6 +128,11 @@ func getCPUSiblingList(cpuRes int64) string {
 }
 
 func setOldKubeletConfig(f *framework.Framework, oldCfg *kubeletconfig.KubeletConfiguration) {
+	// Delete the CPU Manager state file so that the old Kubelet configuration
+	// can take effect.
+	err := exec.Command("/bin/sh", "-c", "rm -f /var/lib/kubelet/cpu_manager_state").Run()
```

Comment:
Could we either make a deleteStateFile helper or move the state path to a constant? Would prefer for that string to live in one place.

Reply:
I moved it to a helper func.